tools/ovc/openvino/tools/ovc/cli_parser.py

```diff
@@ -1033,7 +957,7 @@ def check_bool(value):
 
 def depersonalize(value: str, key: str):
     dir_keys = [
-        'output_dir', 'extension', 'saved_model_dir', 'tensorboard_logdir', 'caffe_parser_path'
+        'extension', 'saved_model_dir', 'tensorboard_logdir', 'caffe_parser_path'
```

Comment:
why do we still mention 'saved_model_dir', 'tensorboard_logdir', 'caffe_parser_path'?

Reply:
Please check in other files by key words like caffe, mxnet, kaldi, tensorboard, etc.

Reply:
Removed 'saved_model_dir', 'tensorboard_logdir', 'caffe_parser_path' from this list.
Checked caffe, mxnet, kaldi, tensorboard in other files, no other mentions were found.